lib/finch.ex

```diff
@@ -231,6 +236,13 @@ defmodule Finch do
           {:ok, acc} | {:error, Exception.t()}
         when acc: term()
   def stream(%Request{} = req, name, acc, fun, opts \\ []) when is_function(fun, 2) do
+    {:ok, config} = Registry.meta(name, :config)
+
+    req =
+      if config.request_transformer,
+        do: config.request_transformer.transform(req, opts),
```

Comment:
Not to belabor this PR any further, but I have been asking myself if there's also value in passing `name` through as well, or is that already something you could infer from the `Request` struct?

Reply:
I don't think there's a way to infer name, so if you think it would be useful now would be the time to add it. Any specific use cases you can think of?

Reply:
I was just thinking maybe you transform differently somehow based on which Finch it is, e.g. what downstream service you’re calling. Now that I think about it though... maybe you'd never need to really do that because you could specify a different `request_transformer` for each Finch supervisor at setup time if you wanted to do that, rather than pointing them all at the same function that dispatches based on name. It's really the same thing I think. 🤷

Reply:
It's easy enough to add (and to ignore if you don't need it) that I went ahead and did it. Open to feedback on whether to keep it there or not.